src/cairo_runner.rs

```diff
@@ -422,4 +488,47 @@ mod test {
             )
         });
     }
+
+    #[test]
+    fn run_from_entrypoint_without_args() {
+        let mut runner = PyCairoRunner::new(
+            "cairo_programs/not_main.json".to_string(),
+            "main".to_string(),
+            Some("plain".to_string()),
+        )
+        .unwrap();
+
+        // Without `runner.initialize()`, an uninitialized error is returned.
+        // With `runner.initialize()`, an invalid memory assignment is returned...
+        //   Maybe it has to do with `initialize_main_entrypoint()` called from `initialize()`?
```

Comment:
While this would certainly be a bug (no builtins are in use), I'd like to see what happens if the layout is `all`.